piptools/scripts/compile.py

```diff
@@ -319,10 +320,14 @@ def cli(
             constraints.extend(reqs)
         elif is_setup_file:
             setup_file_found = True
+            build_runner = quiet_subprocess_runner
+            if verbose:
+                build_runner = default_subprocess_runner
             try:
                 metadata = project_wheel_metadata(
                     os.path.dirname(os.path.abspath(src_file)),
                     isolated=build_isolation,
+                    runner=build_runner,
```

Comment:
```suggestion
            try:
                metadata = project_wheel_metadata(
                    os.path.dirname(os.path.abspath(src_file)),
                    isolated=build_isolation,
                    runner=(
                        default_subprocess_runner
                        if verbose
                        else quiet_subprocess_runner
                    ),
```

Trenary `if` perfectly fits here.

Reply:
I appreciate your suggestion @atugushev.
I consider it opinionated, and personally, I consider using ternary operators less readable and making code harder to maintain (for example, if the number of runners changes, it will require rewriting of a bigger part of the code). 
But since I'm not a maintainer, I'll accept your suggestion! :)